File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        // Create a string containing date and time, BUG FIXED: wrong type
        String dateStr = "2022-03-17 10:45:30";

        // Create DateTimeFormatter object with string, BUG FIXED: missing semi-colon
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");

        // New date object with date string parsed
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);

        // Create new string from the date object
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));

        // Print the new string, BUG FIXED: missing out
        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
from datetime import datetime

# Define string containing date and time in compatible format with date time object
date_str = "2022-03-17 10:45:30

# Create date time object by parsing the string
date_obj = datetime.strptime(date_str '%Y-%m-%d %H:%M:%S')

# Convert the date time object to a string
formatted_date = date_obj.strftime('m/d/Y H:M:S')

# Print date time object
print(formatted_date)
